### ISSUE TITLE
Understanding Loss Functions in Machine Learning

### ISSUE THREAD

**Issue by bloodprashure** [section-engineering-education/engineering-education#832]
### Brief Summary: 
Loss functions, paired up with optimizers is basically how machine learning classifiers and neural networks "learn" from their errors. The concept of a loss function has diversified since the usual mean squared error into many more complicated yet interesting mathematical concepts. This article is an attempt to engage the learner to understand the math behind loss functions and to invoke them in Python.

### Key Takeaways: 
1. To be able to understand the role of loss functions
2. Explore different types of loss functions and their appropriate use case
3. Invoking the loss functions in Python

### References: 
N/A

**Comment by ninjaginja** [comment]
good topic @bloodprashure - approved

**Comment by ninjaginja** [comment]
Closing due to inactivity. If you are still working through this content, you may request to reopen this issue once you are ready to submit your final draft for review via a pull request.

**Comment by bloodprashure** [comment]
Hey Molly, apologies. I had my Semester examinations, hence couldn't submit it. I'm ready with a PR.

**Comment by bloodprashure** [comment]
Hey Molly, please reopen this issue so that I could create a PR.
